packages/@react-spectrum/menu/src/Menu.tsx

```diff
@@ -47,6 +47,8 @@ function Menu<T extends object>(props: SpectrumMenuProps<T>, ref: DOMRef<HTMLULi
   let trayContainerRef = useRef(null);
   let state = useTreeState(completeProps);
   let {menuProps} = useMenu(completeProps, state, domRef);
+  let submenuRef = useRef(null);
+  let style = useSafelyMouseToSubmenu({submenuRef, isOpen: state.expandedKeys.size > 0});
```

Comment:
useSafelyMouseToSubmenu probably should just return a boolean property, then styling can be applied based on that in the render

that or it should return an object with style already on it and that should be merged with other style objects

Reply:
I would lean towards returning the style props so that it is implemented correctly. Although I could see the benefit of returning a boolean if someone wanted to do other things besides just disable pointer events on the menu (although I can't think of any use cases off the top of my head.)

I'll stick with the style object for now, and if someone comes up with a use case, I'll change this. I'll bring it up with the team.

Reply:
it's hard to change an API and if we export this, then it gets locked in, lets bring this up in a standup or otter time

Reply:
I like the style object (or menuProps object e.g `menuProps: {style: {...}}`) since the boolean would require the user to know what to do with it. I'd lean towards menuProps to conform with other hooks that return styles (color/slider/tablecolumnresize hooks)